autoinit/initializer/output_distribution_factory.py

```diff
@@ -114,6 +115,8 @@ def __init__(self, estimator_config: Dict):
             tfkeras.layers.ZeroPadding3D          : ZeroPaddingOutputDistributionEstimator,
             WeightedSum                           : WeightedSumOutputDistributionEstimator,
         }
+        # Supported layers include the builtin layers and the user-provided ones.
+        self.supported_layers = {**self.builtin_layers, **custom_distribution_estimators}
```

Comment:
Just merge the two dictionaries

Reply:
Maybe put this comment in the code.
The python syntax you are using is not very obvious.

Reply:
Good call, thx